src/leader_confirmation_service.rs

```diff
@@ -22,15 +21,15 @@ pub enum ConfirmationError {
 pub const COMPUTE_CONFIRMATION_MS: u64 = 100;
 
 pub struct LeaderConfirmationService {
-    thread_hdl: JoinHandle<()>,
+    current_bank: Arc<Mutex<Option<Arc<Bank>>>>, // super ugly!!!
```

Comment:
If clippy allows this, we need to fire clippy.

Reply:
yea, really bad :).  i think this leader confirmation service needs to be killed completely.  but I think we can do that later

Reply:
Yeah, you can kill it, if in the way. It's not critical and lacks precision.

Reply:
@garious did some work to unhack that